versions/3.0.md

```diff
@@ -2280,7 +2280,7 @@ components:
       href: 'https://na2.gigantic-server.com/#/paths/~12.0~1repositories~1{$response.body#/username}'
```

Comment:
I believe this should be `operationRef:` not `href:` - @darrelmiller please confirm.